test-suite-graal/build.gradle

```diff
@@ -0,0 +1,23 @@
+plugins {
+    id "io.micronaut.build.internal.common"
+    alias(templateLibs.plugins.micronaut.application)
+}
+
+repositories {
+    mavenCentral()
+    maven { url "https://s01.oss.sonatype.org/content/repositories/snapshots/" }
```

Comment:
remove snapshot repository

Reply:
All test modules (`test-aws`, `test-buildtool`, `test-cli` etc.) are using the snapshot repository (via `io.micronaut.internal.starter.base-module` plugin). That is required because `$micronautVersion` is used in 'dependencies' gradle blocks and `$micronautVersion` is `4.0.4-SNAPSHOT`.

For example:
- `build.gradle` from `starter-core` contains: `api platform("io.micronaut.platform:micronaut-platform:$micronautVersion")`

- `io.micronaut.internal.starter.convention` plugin (used by all test modules) contains:
```
dependencies {
    annotationProcessor platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
    implementation platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
    annotationProcessor "io.micronaut:micronaut-inject-java"
    testAnnotationProcessor platform("io.micronaut.platform:micronaut-platform:$micronautVersion")
    testAnnotationProcessor "io.micronaut:micronaut-inject-java"
} 
```

So, usage of the snapshot repository is required in the `test-suite-graal` module too.